CHANGELOG.md

```diff
@@ -3,6 +3,39 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [Unreleased] - ReleaseDate
+### Added
+
+- Added `SockaddrStorage::{as_unix_addr, as_unix_addr_mut}`
```

Comment:
Worth noting that I opted to keep this part of the change, even though it's an API addition rather than simple bugfix.